src/simsoptpp/biot_savart_impl.h

```diff
@@ -161,10 +163,254 @@ void biot_savart_kernel(AlignedPaddedVec& pointsx, AlignedPaddedVec& pointsy, Al
     }
 }
 
+#else
+
+template<class T, int derivs>
+void biot_savart_kernel_openmp(AlignedPaddedVec& pointsx, AlignedPaddedVec& pointsy, AlignedPaddedVec& pointsz,
```

Comment:
Is this only called in the profiling code?

Reply:
Yes. Since this is not used in production code, I deleted this.